### ISSUE TITLE
Training goes on infinitely

### ISSUE THREAD

**Issue by naveen-kinnal** [cindyxinyiwang/TextStyleTransfer#16]
Hello,

I see that the training for the YELP dataset goes on infinitely since the training in main.py has the condition 'while true'. May I know how many epochs should we train?

**Comment by jxhe** [comment]
Hi,

The training will end automatically when the val loss stops decreasing several times, as controlled here: 

https://github.com/cindyxinyiwang/deep-latent-sequence-model/blob/8a798582b1af5ef7f6ac4ca1f2138fd382a1cb06/src/main.py#L743 

You can reduce the `patience` number to make it end more quickly

**Comment by naveen-kinnal** [comment]
Thank you. I have another question

1. Why is the 'dev_src_file' and 'dev_trg_ref' the same? 
2. If the src_file contains the negative sentences, the trg_file will contain the negative attributes?

**Comment by jxhe** [comment]
Can you clarify where "dev_src_file" and "dev_trg_ref" are?
